Extensions/Lighting/lightruntimeobject.js

```diff
@@ -0,0 +1,93 @@
+gdjs.LightRuntimeObject = function (runtimeScene, lightObjectData) {
+  gdjs.RuntimeObject.call(this, runtimeScene, lightObjectData);
+
+  this._radius =
+    lightObjectData.content.radius > 0 ? lightObjectData.content.radius : 1;
+  this._color = gdjs.LightRuntimeObject.hexToRGBColor(
+    lightObjectData.content.color
+  );
+  this._debugMode = lightObjectData.content.debugMode;
+  this._texture =
+    lightObjectData.content.texture === ''
+      ? null
+      : runtimeScene
+          .getGame()
+          .getImageManager()
+          .getPIXITexture(lightObjectData.content.texture);
+
+  if (gdjs.LightObstaclesManager)
```

Comment:
Is there even a reason for gdjs.LightObstaclesManager not to be defined? I don't think so? We should always include I think the behavior with the object in the include files.

Reply:
> Is there even a reason for gdjs.LightObstaclesManager not to be defined

I'm not sure why this happens but whenever there's no object with the obstacle behavior, the game crashes saying gdjs.LightObstaclesManager is undefined.